packages/ember-metal/tests/mixin/merged_properties_test.js

```diff
@@ -91,6 +91,28 @@ test('mergedProperties should be concatenated', function() {
   deepEqual(get(obj, 'bar'), { a: true, l: true, e: true, x: true }, "get bar");
 });
 
+test("mergedProperties should exist even if not explicitly set on create", function(){
```

Comment:
JSCS is complaining (and failing the build) on the missing space before the curly bracket.  Needs to be `function() {`.